archive/p/python/base64_encode_decode.py

```diff
@@ -0,0 +1,40 @@
+import base64
+import binascii
+import sys
+from typing import NoReturn
+
+
+def usage() -> NoReturn:
```

Comment:
I have never actually seen this before. Is this because the function exits no matter what, so it has no chance to return `None`?

Reply:
Yes, `NoReturn` means what it says: the function never returns. The program just exits.